mod/wikirate/set/type/claim.rb

```diff
@@ -94,7 +94,7 @@ def next_step_tip
     elsif (not companies = Card["#{card.name}+company"]) || companies.item_names.empty?
       'improve this note by adding a company.'
     else
-      cited_in = Card.search :refer_to => card.name, :left=>{:type_id=>WikirateAnalysisID}, :right=>{:name=>Card[:wikirate_article].name}
+      cited_in = Card.search :refer_to => card.name, :left=>{:type_id=>WikirateAnalysisID}, :right=>{:name=>Card[:overview].name}
```

Comment:
Line is too long. [129/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.